Source/BestProjectEver/Public/AI/BPE_PatrolPath.h

```diff
@@ -0,0 +1,33 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameFramework/Actor.h"
+#include "BPE_PatrolPath.generated.h"
+
+class USplineComponent;
+
+UCLASS()
+class BESTPROJECTEVER_API ABPE_PathFollowing : public AActor
+{
+	GENERATED_BODY()
+	
+public:	
+
+	ABPE_PathFollowing();
+
+protected:
+
+	/** point the enemy will follow */
+	UPROPERTY(EditAnywhere, Category="Components")
```

Comment:
Does this component need to be Editable? Couldn't it be marked as Visible?

Reply:
Yess sorry I thought it was necesary to create the points on the map 
![image](https://user-images.githubusercontent.com/85327912/173104511-7fb8d736-ba8f-42f4-af90-7315d40f83b4.png)